test/Microsoft.AspNetCore.Razor.Test/TestFiles/CodeGenerator/Output/PrefixedAttributeTagHelpers.Reversed.cs

```diff
@@ -220,15 +198,11 @@ public PrefixedAttributeTagHelpers()
             {
                 throw new InvalidOperationException(FormatInvalidIndexerAssignment("string-prefix-thyme", "TestNamespace.InputTagHelper1", "StringDictionaryProperty"));
             }
-#line 22 "PrefixedAttributeTagHelpers.cshtml"
-__TestNamespace_InputTagHelper2.IntDictionaryProperty["value"] = 37;
-
-#line default
-#line hidden
-            __tagHelperExecutionContext.AddTagHelperAttribute("int-prefix-value", __TestNamespace_InputTagHelper2.IntDictionaryProperty["value"]);
+            __TestNamespace_InputTagHelper2.IntDictionaryProperty["value"] = __tagHelperAttribute_12.Value.ToString();
+            __tagHelperExecutionContext.AddTagHelperAttribute(__tagHelperAttribute_12);
             __TestNamespace_InputTagHelper1.IntDictionaryProperty["value"] = __TestNamespace_InputTagHelper2.IntDictionaryProperty["value"];
-            __TestNamespace_InputTagHelper2.StringDictionaryProperty["thyme"] = "string";
-            __tagHelperExecutionContext.AddTagHelperAttribute("string-prefix-thyme", __TestNamespace_InputTagHelper2.StringDictionaryProperty["thyme"]);
+            __TestNamespace_InputTagHelper2.StringDictionaryProperty["thyme"] = __tagHelperAttribute_13.Value.ToString();
```

Comment:
For string typed, there will be no allocations of new strings when we do ToString()